src/main/java/uk/gov/hmcts/reform/sandl/snlrules/utils/DateTimeUtils.java

```diff
@@ -1,17 +1,27 @@
 package uk.gov.hmcts.reform.sandl.snlrules.utils;
 
+import org.apache.commons.lang3.time.DurationFormatUtils;
 import uk.gov.hmcts.reform.sandl.snlrules.exception.DateComparisonException;
 
+import java.time.Duration;
 import java.time.LocalDateTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
 import java.time.ZoneOffset;
 import java.time.format.DateTimeFormatter;
 import java.time.temporal.ChronoUnit;
 
 public final class DateTimeUtils {
+
+    private static ZoneId zone = ZoneOffset.systemDefault();
```

Comment:
it's everything fine with this code, but I'm wondering if we can use here Lombok setter?

Reply:
ass talked with Kamil P. assumption is that Rules will be in UK time zone, events should also, and frontend is sending dateTime formatted to zone 0 (if that changes it should include zone in it's string).